src/main/java/com/wynntils/modules/utilities/overlays/inventories/HotbarChargeOverlay.java

```diff
@@ -0,0 +1,39 @@
+package com.wynntils.modules.utilities.overlays.inventories;
+
+import com.wynntils.McIf;
+import com.wynntils.core.events.custom.RenderEvent;
+import com.wynntils.core.framework.interfaces.Listener;
+import com.wynntils.modules.utilities.configs.UtilitiesConfig;
+import net.minecraft.init.Items;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+public class HotbarChargeOverlay implements Listener {
+
+    @SubscribeEvent
+    public void onHotbarOverlay(RenderEvent.DrawItemOverlay event) {
+        if (!UtilitiesConfig.INSTANCE.showConsumableChargesHotbar) return;
+        ItemStack is = event.getStack();
+        Item item = is.getItem();
+
+        // We don't care about items inside the inventory, only in the hotbar
+        if (McIf.player().inventory.mainInventory.contains(is) && McIf.player().inventory.mainInventory.indexOf(is) > 8) return;
+        if (!(item == Items.POTIONITEM) || item == Items.DIAMOND_AXE) return; // Consumables are only potions or diamond axes (crafted)
```

Comment:
```suggestion
        if (item != Items.POTIONITEM) && item != Items.DIAMOND_AXE) return; // Consumables are only potions or diamond axes (crafted)
```

Isn't this what you meant to do?

Reply:
Yup thanks lol, I just copied directly from the previous file without really looking at it too much